ui/src/criteria/concept.tsx

```diff
@@ -23,32 +25,46 @@ type Selection = {
   name: string;
 };
 
-export class ConceptCriteria extends Criteria {
-  constructor(name: string, public filter: string) {
-    super(name);
+type Data = {
+  filter: string;
+  selected: Selection[];
+};
+
+@registerCriteriaPlugin("condition", "Condition", "Contains Conditions Codes")
```

Comment:
This will be necessary long term. Once the entity model and UI hints are fully realized, this will become more generic and some or all of the concept domains will be handled by a single plugin but there will be at least one more for demographics in the core repo. Further down the road I expect additional plugins for things like documents, genomes, and FitBit data. Beyond even those, this is a key point of extensibility where users will need to integrate custom criteria, particularly if they integrate with external data sources.

I also think there's potential here to have partial plugins or extensions where users want to customize the UI in some fashion over standard data but I haven't thought about that too much yet.